CHANGELOG.md

```diff
@@ -39,6 +39,7 @@
 - Added shortcodes for [fork-awesome icons](https://forkaweso.me/Fork-Awesome/icons/) (e.g. `:fa-picture-o:`)
 - The code button now adds code fences even if the user selected nothing beforehand
 - Code blocks with 'csv' as language render as tables.
+- All images can be clicked to show them a bit bigger.
```

Comment:
"a bit bigger"?
A bit unprecise. How about something like "to show them in full screen"?

Reply:
done